.gitignore

```diff
@@ -1,2 +1,10 @@
-/target
-/project/target
+target
+
+# IDE #
+######################
+nbproject
+.idea
+nbactions.xml
+*.iml
+private
+classes
```

Comment:
(some of those you could add to your global gitignore so you don't need them in each project, https://help.github.com/en/github/using-git/ignoring-files#create-a-global-gitignore) - but they're :+1: here as well

Reply:
yes, unsure why i did this, actually, i have the same set up with a global gitignore: leaving `target` only.